docs/cluster_and_device_type_dev/index.md

```diff
@@ -0,0 +1,13 @@
+# Cluster and Device Type development
+
+The following docs present a guide on how to develop new clusters and device
+types in the SDK.
+
+```{toctree}
+:glob:
+:maxdepth: 1
+:hidden:
+
+```
```

Comment:
This seems to render as well, is that intentional?

Reply:
it renders on the other index files too, but not in the actual site. Not sure why.

Reply:
Properly just a github quirk then. 🤷🏻‍♂️